doc/cascadia/profiles.schema.json

```diff
@@ -0,0 +1,449 @@
+{
+  "$id": "https://github.com/microsoft/terminal/terminal.profile.schema.json",
```

Comment:
I don't know what the `$id` is supposed to represent, but this is not a valid URL